File: code_PHP/fibonacciSeries.php
<?php
$number = readline("Enter the number of elements: ");

$n1 = 0;
$n2 = 1;

echo $n1 . " " . $n2 . " ";

for ($i = 2; $i < $number; $i++) {
    $n3 = $n1 + $n2;
    echo $n3 . " ";
    $n1 = $n2;
    $n2 = $n3;
}
?>

File: code_Rust/fibonacci_series.rs
use std::io;

fn main() {
    println!("Enter the number of elements:");
    let mut input = String::new();
    io::stdin().read_line(&mut input).expect("Failed to read line");
    let number: u32 = input.trim().parse().expect("Invalid input");

    let (mut n1, mut n2) = (0, 1);
    print!("{} {} ", n1, n2);

    for _ in 2..number {
        let n3 = n1 + n2;
        print!("{} ", n3);
        n1 = n2;
        n2 = n3;
    }
}